config-keyvault/pom.xml

```diff
@@ -5,23 +5,23 @@
     <modelVersion>4.0.0</modelVersion>
     
     <parent>
-        <groupId>com.azure</groupId>
-        <artifactId>microprofile-parent</artifactId>
+        <groupId>com.azure.microprofile</groupId>
+        <artifactId>azure-microprofile-parent</artifactId>
         <version>1.0.0</version>
         <relativePath>../parent/pom.xml</relativePath>
     </parent>
     
-    <groupId>com.azure</groupId>
-    <artifactId>microprofile-config-keyvault</artifactId>
+    <groupId>com.azure.microprofile</groupId>
+    <artifactId>azure-microprofile-config-keyvault</artifactId>
     <version>1.0.0</version>
```

Comment:
This is a GA version. Do we want to release preview versions first to get feedback from customers before releasing a GA? If yes, we should use `1.0.0-beta.1`.

Reply:
Good point, I think it's good to release preview versions first. Fixed in 8bf0a1d.